website/components/features-list/style.module.css

```diff
@@ -0,0 +1,107 @@
+.featureListContainer {
+  background-color: #000;
+  padding-top: 128px;
+  background-position: right top, left bottom;
+  background-repeat: no-repeat;
+  color: #fff;
+}
+
+.featureContainer {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  & .featureTextContainer {
+    display: flex;
+  }
+
+  & .imageContainer {
+    max-width: 490px;
+    margin: 0 auto;
+    padding-bottom: 40px;
+
+    & img {
+      max-width: 100%;
+    }
+  }
+
+  @media (--large) {
+    flex-wrap: nowrap;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+
+    & .featureText {
+      width: 488px;
+    }
+
+    & .imageContainer {
+      margin: 0;
+    }
+  }
+}
+
+.featuresContainer {
+  padding-top: 157px;
+  padding-bottom: 394px;
+  display: grid;
+  row-gap: 120px;
```

Comment:
This is a really subtle, but excellent decision.

Often, I'll see the decision to put spacing on the *feature* item (as margin-top/margin-bottom), which is less preferable to than what you did here, as the spacing between each feature is a contextual decision to the fact they are in a list.  You approach sets us up for success if we ever need to show one of these feature cards in a different context.

Great stuff ❇️  definitely impressed.

Reply:
Yeah, this way we don't have to remove the margin on the last feature, etc. Thanks!